geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/collections/SizeableByteArrayList.java

```diff
@@ -141,6 +141,20 @@ public boolean remove(Object o) {
     return element;
   }
 
+  @Override
+  public byte[] set(int index, byte[] newElement) {
+    byte[] replacedElement = super.set(index, newElement);
+    memberOverhead -= calculateByteArrayOverhead(replacedElement);
+    memberOverhead += calculateByteArrayOverhead(newElement);
+    return replacedElement;
+  }
+
+  @Override
+  public void add(int index, byte[] element) {
+    memberOverhead += calculateByteArrayOverhead(element);
+    super.add(element);
```

Comment:
I just noticed that this should be `super.add(index, element);` which is why the DUnit tests are failing in pre-checkin.